### PR TITLE
Adds info about python wheels.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

## Summary
 - Adds info about maliput python wheels

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: docs/maliput_wheels_pypy_index.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+***************************
+
+.. contents:: Table of Contents
+    :depth: 5
+
+
+Python Wheels
+=============
+
+As indicated in the :ref:`installation_label` section, `maliput` is supported to be installed via `pip`.
```
Comment (agalbachicar) [inline docs/maliput_wheels_pypy_index.rst:14]:
```suggestion
As indicated in the :ref:`installation_label` section, `maliput`_ is supported to be installed via `pip`_.
```

Comment (agalbachicar) [reply docs/maliput_wheels_pypy_index.rst:14]: Am I right? If so, then I think we are missing the _ in various places.

File: docs/maliput_wheels_pypy_index.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+
+An example of this is the `maliput_malidrive`_ package, which provides an OpenDRIVE-based implementation of the `maliput`_ API.
+Its entry point method is defined at https://github.com/maliput/maliput_malidrive/blob/main/maliput_malidrive/__init__.py so that it can be discovered by `maliput`_.
+
+Wheel Generation
+^^^^^^^^^^^^^^^^
+
+Following `manylinux2014_x86_64` guidelines, the `maliput` wheels are built on a CentOS 7 docker image: quay.io/pypa/manylinux2014_x86_64.
+A custom docker image is used to build the wheels, and the dockerfile is defined at `maliput_infrastructure`_ repository where a GitHub Action is defined build and push the docker image to GHCR.
```
Comment (agalbachicar) [inline docs/maliput_wheels_pypy_index.rst:40]: Can we add a link here?

**Review by agalbachicar — CHANGES_REQUESTED** [review]
PTAL.

File: docs/maliput_wheels_pypy_index.rst (hunk 1 above)
Comment (francocipollone) [reply docs/maliput_wheels_pypy_index.rst]: By using `my_link`_ I reference this to a link mentioned in this page (see the bottom of the file).
I added a couple of missing _ however when I refer to maliput as the entire ecosystem (e.g. `maliput` maintainers) it doesn't make sense to have a link to maliput package.

File: docs/maliput_wheels_pypy_index.rst (hunk 2 above)
Comment (francocipollone) [reply docs/maliput_wheels_pypy_index.rst]: Good point.
Done!

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**